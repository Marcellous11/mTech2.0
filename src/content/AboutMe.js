import React from 'react';
import '../styles/AboutMe.css';
const AboutMe = () => {
	return (
		<div className="AboutMe">
			<div className="AboutMe-Intro">
				<p className="Title">About Me</p>
				<ul>
					<li>
						Full-Stack <b>Web Developer</b> and <b>Software Engineer</b>
					</li>
					<li>{`Born in Las Vegas, NV --> now in Frisco, TX`}</li>
					<li>Hobbies include: Watching movies, rock climbing and playing soccer!</li>
				</ul>
			</div>
		</div>
	);
};
export default AboutMe;
