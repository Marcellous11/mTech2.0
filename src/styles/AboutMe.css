.AboutMe-Intro p {
	color: rgb(0, 230, 0);
}

.AboutMe-Intro ul li {
	color: rgb(0, 230, 0);
	line-height: 1.5em;
}

@media only screen and (max-width: 830px) {
	.AboutMe-Intro ul li {
		margin-left: -1.5em;
	}
}
