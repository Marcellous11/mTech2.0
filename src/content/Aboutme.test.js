import { render, screen } from '@testing-library/react';
import AboutMe from './AboutMe';

test('Smoke test, does the component render', () => {
	render(<AboutMe />);
});

test('Snapshot ', () => {
	const { asFragment } = render(<AboutMe />);

	expect(asFragment()).toMatchSnapshot();
});

test('should have Quick facts about me', () => {
	const { getByText } = render(<AboutMe />);

	expect(getByText('Quick facts about me')).toBeInTheDocument();
});
