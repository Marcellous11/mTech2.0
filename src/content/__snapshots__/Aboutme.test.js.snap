// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot  1`] = `
<DocumentFragment>
  <div
    class="AboutMe"
  >
    <div
      class="AboutMe-Intro"
    >
      <p
        class="Title"
      >
        Quick facts about me
      </p>
      <ul>
        <li>
          Full-Stack Web Developer and Software Engineer
        </li>
        <li>
          Born in Las Vegas, NV --&gt; now in Frisco, TX
        </li>
        <li>
          Habbies include: Watching moives, rock climbing and playing soccer!
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
